source/client/HandView.py

```diff
@@ -30,15 +36,19 @@ def __init__(self, controller, display):
         self.discard_confirm = False
         self.num_wilds = 0
         self.wild_cards = []
+        self.selected_list = []
         self.round_index = 0
         self.round_advance = False
-        self.round_meld = Meld_Threshold  #[50,90,120,150]
-        self.betweenrounds = ['Welcome to a new game.  This is the round of 50.',\
-                        'To draw click on the deck of cards (upper left).',\
-                        'To discard select ONE card & double click on discard button. ',\
-                        'To pick up pile PREPARE necessary cards & then click on discard pile. ',\
-                        "Cumulative score will display beneath player's cards",\
-                        'When ready to start playing click on the YES button on the lower right.']
+        self.round_meld = Meld_Threshold  # [50,90,120,150]
```

Comment:
This shouldn't need to be a part of the hand_view anymore, just use the Meld_Threshold

Reply:
Done.